DataFormats/BTauReco/interface/TemplatedSecondaryVertexTagInfo.h

```diff
@@ -58,7 +58,7 @@ class TemplatedSecondaryVertexTagInfo : public BaseTagInfo {
 
         struct VertexData {
                 VTX                             vertex;
-                Measurement1D                   dist2d, dist3d;
+                Measurement1D                   dist1d,dist2d, dist3d;
                 GlobalVector                    direction;
 		
 		// Used by ROOT storage
```

Comment:
as I noticed in the previous comment, the class version should be updated in this file
(TemplatedSecondaryVertexTagInfo.h)